### DISCUSSION TITLE
Conditional loading of packages – a bug?

### DISCUSSION THREAD

**Discussion by kellertuer** [fonsp/Pluto.jl#2345] [Questions about using Pluto]
I have a question about conditional loading of packages.
Short background: We have a Pluto notebook with interactive parts that we also want to include in the docs (using PlutoStaticHTML). For that we have a Boolean to toggle to “static mode” of the notebook, since we can not use WGLMakie.jl in the CI. So we only load that if the toggle is true.

If I recreate that scenario (with our package as an example, which defines `is_point`) on Julia REPL, I get

```Julia-repl
❯ julia
               _
   _       _ _(_)_     |  Documentation: https://docs.julialang.org
  (_)     | (_) (_)    |
   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 1.8.1 (2022-09-06)
 _/ |\__'_|_|_|\__'_|  |  Official https://julialang.org/ release
|__/                   |

julia> a = false
false

julia> a && using Manifolds
false

julia> is_point
ERROR: UndefVarError: is_point not defined

julia> 
```

If I do the same three things in a Pluto Notebook, I get

<img width="1258" alt="Screenshot 2022-10-26 at 20 23 15" src="https://user-images.githubusercontent.com/115566/198105998-2a0e6a1c-1304-4c35-acdf-339b43185d70.png">

even if I do not use the Pluto package manager I get (yes Manifolds is installed in my global environment ;) )

<img width="1258" alt="Screenshot 2022-10-26 at 20 24 30" src="https://user-images.githubusercontent.com/115566/198106264-3a9f1ca7-614b-4c1f-a4cf-2f5780079ba1.png">

So – is that a bug?
Or phrased differently: How can I achieve the result from Julia REPL, conditionally loading a package, in Pluto?

**Comment by fonsp** [comment]
@Pangoraw
